testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/ResetPasswordTest.java

```diff
@@ -1143,34 +1140,6 @@ public void failResetPasswordServiceAccount() {
         assertEquals("Invalid username or password.", errorPage.getError());
     }
 
-    @Test
-    public void resetPasswordLinkNewTabAndProperRedirectAccount() throws IOException {
```

Comment:
@mposolda Just to be sure you are aware this test is removed. I have checked and I think it's more or less testing the same the the next one so I think it's OK. Just double-checking.

Reply:
Yes, exactly. Moreover, this test was not correct and it was relying on the old account console, which we already removed. The test "accidentally" passed before even if old account console was not present, but it did not had much sense as it was was displaying some error pages etc.